pootle/apps/pootle_fs/management/commands/fs_commands/rm.py

```diff
@@ -19,4 +19,4 @@ def add_arguments(self, parser):
             "--force",
             action="store_true",
             dest="force",
-            help=("Stage files and stores that are conflicting for removal"))
+            help=("Stage files and/or stores for removal"))
```

Comment:
the logic is this
- if a store or file has been removed, the rm command will stage the corresponding resource for removal
- if used with force, it will also stage *any* conflicting stores/files and *any* new stores/files